tests/ord-aggregator/tests/handler_test.go

```diff
@@ -327,6 +330,18 @@ func TestORDAggregator(stdT *testing.T) {
 		defer fixtures.CleanupApplication(t, ctx, certSecuredGraphQLClient, testConfig.DefaultTestTenant, &seventhApp)
 		require.NoError(t, err)
 
+		k8sClient, err := clients.NewK8SClientSet(ctx, time.Second, time.Minute, time.Minute)
+		require.NoError(t, err)
+		jobName := "ord-aggregator-test"
+		namespace := "compass-system"
+		k8s.CreateJobByCronJob(t, ctx, k8sClient, "compass-ord-aggregator", jobName, namespace)
```

Comment:
Here you spawn the ord aggregator job on-demand, but if there is already an existing ord aggregator job running then we don't know how both of them might collide. Additionally even if there was no other parallel job already running, you might hit the case where this e2e tests finishes and tries to cleanup it's created app resources but just at that time a new ord aggregator regular job spawns naturally and locks the app resources that you want to delete which will block/slow down the test until the ord aggregator job finishes.

That's why you need to ensure that for the duration of this e2e test that the regular ord aggregator job doesn't interfere.

Here's a suggestion, but if you have a better idea, we can discuss:
- add a new configuration in values.yaml called `ordAggregator.suspend`:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/15074116/186363760-76dd9a67-8385-4511-93fc-327726eb9f3e.png">
- Then use it in the ord aggregator job yaml:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/15074116/186364103-51d936d6-26b6-4fff-bcf9-59181636a467.png">
- The `suspend` value will be `true` by default as it is in the picture, but we'll make sure to override it to `false` in the mgt-plane-config repo for our real environments. This way we can disable the ord aggregator for local scenarios and for our dev-e2e-tests, but have it running regularly (not suspended) for our real envs

Reply:
The only down-side to this approach is that if someone develops sth about the ord aggregator on his local machine, he needs to be aware that the ord aggregator will be suspended by default. He'll have to either enable it manually, or just trigger the cronjob manually (as you do in the tests) whenever he wants to check something with the aggregator.

I think this is a smaller price to pay than the alternative that I can think of - the alternative is that we programatically suspend the ord aggregator job from the e2e test code by patching the ord aggregator. But this has the bigger (in my opinion) downside of the ord aggregator potentially runnning at the same time - we'll have to wait for it to finish before the test can suspend the job and then start running the test cases.